common/src/main/java/com/mapbox/vision/common/models/ArFeature.kt

```diff
@@ -0,0 +1,31 @@
+package com.mapbox.vision.common.models
+
+sealed class ArFeature {
+
+    open val caption: String = ""
```

Comment:
Can this 3 vals just be params in constructor, should be shorter then, eg.
```
sealed class ArFeature(
  val caption: String,
  ...
) {
  object Lane : ArFeature(
     caption = "blabla",
     ...
  )
}
```

Reply:
Yeah, reasonable.